### ISSUE TITLE
Button for the toolbar to change/cycle through Themes

### ISSUE THREAD

**Issue by AmeJunn** [d2phap/ImageGlass#2010]
### Describe the feature
I often find myself changing between my light and dark themes depending on the image, to see it in a different context. But it's really annoying to go to the settings every time to change the theme. It would be nice if there was a quick button, able to be placed on the toolbar, or via hotkey, to quickly change the theme.

### How to use feature
1. Click the button / press the hotkey
2. The theme is changed

### Additional context
Perhaps the themes could be marked to be changed via the button. I use 3 themes, so it's not really a on/off toggle for me for example.

But even if it's just a Dark mode/Light mode switch (like in many websites) it would still be nice. I could set the dark mode theme and the light mode theme and then just press the switch.

**Comment by d2phap** [comment]
Similar to https://github.com/d2phap/ImageGlass/issues/2042#issuecomment-2480449602, in the upcoming v9.2, you can use new API `IG_ApplySettings(string json)` to change the settings without using Settings window.

Here is the JSON to add "Change theme" button to toolbar, in the `Arguments`, you just need to pass the theme pack name for both `DarkTheme` and `LightTheme`:


Change to Dark theme:
```json
{
  "Type": "Button",
  "Id": "btn_custom_1",
  "Text": "Change to Dark theme",
  "DisplayStyle": "Image",
  "CheckableConfigBinding": "",
  "Alignment": "Left",
  "Image": "C:\\path\\to\\icon.svg",
  "OnClick": {
    "Executable": "IG_ApplySettings",
    "Arguments": [
      "{ \"DarkTheme\": \"Kobe\", \"LightTheme\": \"Kobe\" }"
    ]
  }
}
```

Change to Light theme:
```json
{
  "Type": "Button",
  "Id": "btn_custom_2",
  "Text": "Change to Light theme",
  "DisplayStyle": "Image",
  "CheckableConfigBinding": "",
  "Alignment": "Left",
  "Image": "C:\\path\\to\\icon.svg",
  "OnClick": {
    "Executable": "IG_ApplySettings",
    "Arguments": [
      "{ \"DarkTheme\": \"Kobe-Light\", \"LightTheme\": \"Kobe-Light\" }"
    ]
  }
}
```

![image](https://github.com/user-attachments/assets/b9fbad28-fe1f-4865-8a75-5c51f03ff5a9)

**Comment by d2phap** [comment]
Updated: You can assign hotkeys to the toolbar buttons, too!

```json
{
  "Type": "Button",
  "Id": "btn_custom_1",
  "Text": "Change to Dark theme",
  "DisplayStyle": "Image",
  "CheckableConfigBinding": "",
  "Alignment": "Left",
  "Image": "C:\\path\\to\\icon.svg",
  "OnClick": {
    "Executable": "IG_ApplySettings",
    "Arguments": [
      "{ \"DarkTheme\": \"Kobe\", \"LightTheme\": \"Kobe\" }"
    ]
  },
  "Hotkeys": ["Alt+D"]
}
```

![image](https://github.com/user-attachments/assets/ea1498bb-c12e-4994-a651-3eb2fd1719c8)

**Comment by AmeJunn** [comment]
Nice thank you!
I'll try it out when it comes out

**Comment by d2phap** [comment]
Published [ImageGlass 9.2.0.1208 - Maroon 4](https://github.com/d2phap/ImageGlass/releases/tag/9.2.0.1208)
